commerce_coordinator/apps/lms/views.py

```diff
@@ -334,3 +335,24 @@ def post(self, request) -> Response:
             logger.exception(f"[RefundView] Exception raised in {self.post.__name__} with error {repr(e)}")
             return Response('Exception occurred while retiring Commercetools customer',
                             status=HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+class FirstTimeDiscountEligibleView(APIView):
+    """View to check if a user is eligible for a first time discount"""
+    permission_classes = [LoginRedirectIfUnauthenticated]
+    throttle_classes = [UserRateThrottle]
+
+    def get(self, request):
+        """Return True if user is eligible for a first time discount."""
+        email = request.query_params.get('email')
+
+        if not email:  # pragma: no cover
+            raise PermissionDenied(detail="Could not detect user email.")
+
+        result = CheckFirstTimeDiscountEligibility.run_filter(email=email)
+
+        output = {
+            "is_eligible": result.get('is_eligible', True)
```

Comment:
should the default value be True?

Reply:
For the default case, the user is going to be eligible for the first-time discount.